src/index/Index.h

```diff
@@ -356,25 +363,26 @@ class Index {
    */
   CompactStringVector<Id, Id> _hasPredicate;
 
-  size_t passTsvFileForVocabulary(const string& tsvFile);
-
-  void passTsvFileIntoIdVector(const string& tsvFile, ExtVec& data);
-
   // Create Vocabulary and directly write it to disk. Create ExtVec which can be
   // used for creating permutations
   // Member _vocab will be empty after this because it is not needed for index
   // creation once the ExtVec is set up and it would be a waste of RAM
   template <class Parser>
-  ExtVec createExtVecAndVocab(const string& ntFile);
+  std::unique_ptr<ExtVec> createExtVecAndVocab(const string& ntFile);
```

Comment:
I'm not super happy with the whole `ExtVec` naming, it sounds a lot like a "how" not a "why" or "what". In this case can't we just drop it from the method name?

Reply:
The ExtVec is renamed to StxxlVec and the Method now is
`createIdTriplesAndVocab`. Is that better?